site/realm/functions/unlinkReportsFromIncidents.js

```diff
@@ -0,0 +1,8 @@
+exports = async (input) => {
+  
```

Comment:
Let's keep the implementation, but raise an error at present since we don't want to have orphaned reports.